pom.xml

```diff
@@ -14,7 +14,7 @@
     <parent>
         <artifactId>spring-boot-starter-parent</artifactId>
         <groupId>org.springframework.boot</groupId>
-        <version>1.5.22.RELEASE</version>
+        <version>2.4.4</version>
```

Comment:
You probably need to upgrade [spring-cloud.version](https://github.com/thehyve/podium/blob/update/backend/pom.xml#L123) at the same time to a compatible version (check https://spring.io/projects/spring-cloud#learn for the latest)